### PR TITLE
RSVP for events

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/html/ui/post/card.tpl.html [hunk 1]
```diff
@@ -140,7 +140,20 @@
       --><a class="fulfill" ng-click="markFulfilled()" ng-if="isPostOwner() && !post.fulfilled_at && post.type !== 'chat'">
         <i class="icon-little-check"></i>
         <span class="hidden-xs">Mark complete</span>
-      </a>
+      </a><!--
+
+      --><div class="post-controls no-border" ng-if='post.type === "event"'><a class="rsvp-yes" ng-class="eventResponse === 'yes' ? 'rsvp-selected' : ''" ng-click="changeEventResponse('yes')">
+        <span class="hidden-xs">Going</span>
```
Comment (levity) [inline src/html/ui/post/card.tpl.html]: what's going to happen for mobile?

Comment (robbiecarlton) [inline src/html/ui/post/card.tpl.html]: I haven't looked at it yet. This is just a sketch to get the interface working.

**Comment by robbiecarlton** [comment]
screen shot 2015-10-29 at 5 00 02 pm
screen shot 2015-10-29 at 4 59 41 pm
screen shot 2015-10-29 at 4 59 30 pm
screen shot 2015-10-29 at 4 59 21 pm
screen shot 2015-10-29 at 4 59 11 pm

**Comment by robbiecarlton** [comment]
@edwardwest gave the thumbs up for the design, @levity ok to merge?

**Comment by levity** [comment]
i think the icons next to the button labels are actually confusing -- the checkmark kind of suggests that that button is selected and the ? in front of the other one just seems weird. I would vote to omit them entirely.

also, rather than a gray padded box, which pushes the buttons out of alignment with the left edge of the post, why not just have a horizontal rule above them?

those two changes would also create room for the three buttons to be side by side on mobile. did you consider using a [button group](http://getbootstrap.com/components/#btn-groups)?

**Comment by levity** [comment]
also, what happens when the list of RSVP's is long?

**Comment by robbiecarlton** [comment]
screen shot 2015-10-29 at 5 45 16 pm
Change the icons

**Comment by robbiecarlton** [comment]
Something like this: 
screen shot 2015-11-03 at 11 04 21 am

**Comment by robbiecarlton** [comment]
When the list of rsvps is long, it just opens a long list, and extends the page down to the length of the list.

I didn't know about button group. Looks cool. You want me to use it for this?

**Comment by levity** [comment]
Yeah, I think it makes sense here.

**Comment by robbiecarlton** [comment]
screen shot 2015-11-03 at 3 54 46 pm

Good to merge?

**Comment by levity** [comment]
yay! :shipit:

**Pull request merged**